src/Transition/VCompareGlobalRoutines.f90

```diff
@@ -749,14 +790,27 @@ SUBROUTINE CheckSpecialObjects(DifUnit,ObjectName,CurArgs,OutArgs,FieldNames,Fie
           CALL WriteOutIDFLinesAsSingleLine(DifUnit,ObjectName,CurArgs,OutArgs,FieldNames,FieldUnits)
 
         CASE('BUILDINGSURFACE:DETAILED')
-          CALL WriteOutPartialIDFLines(DifUnit,ObjectName,10,OutArgs,FieldNames,FieldUnits)
           IF (MakeUPPERCase(OutArgs(10)) == 'AUTOCALCULATE') THEN
             NVert=(CurArgs-10)/3
           ELSEIF (OutArgs(10) == '') THEN
             NVert=(CurArgs-10)/3
           ELSE
-            READ(OutArgs(10),*) NVert
+            ! Try to read the number of vertices as an integer
+            NVert = ProcessNumber(OutArgs(10),ErrFlag)
+            IF (ErrFlag) THEN
+              ! If failed, default to autocalculate instead
+              NVert=(CurArgs-10)/3
+              ! Replace in existing with Autocalculate
+              OutArgs(10) = 'Autocalculate'
+              CALL DisplayString('For ' // TRIM(ObjectName) // ' named ''' // TRIM(OutArgs(1)) // &
+                  ''', Number of vertices is not an integer, defaulting to Autocalculate (N='// &
```

Comment:
Call ShowWarningError instead of DisplayString so the message goes into the audit output.

Change the message to ". . . vertices is not ~an integer~ *a number*, defaulting . . . "